publication-commons/src/main/java/no/unit/nva/publication/RequestUtil.java

```diff
@@ -58,7 +59,14 @@ public static UUID getFileIdentifier(RequestInfo requestInfo) throws ApiGatewayE
         return Optional.ofNullable(requestInfo.getPathParameters())
                    .map(params -> params.get(FILE_IDENTIFIER))
                    .map(UUID::fromString)
-                   .orElseThrow(() -> new BadRequestException("Could not get file identifier!"));
+                   .orElseThrow(() -> new BadRequestException(COULD_NOT_GET_FILE_IDENTIFIER));
+    }
+
+    public static SortableIdentifier getFileEntryIdentifier(RequestInfo requestInfo) throws ApiGatewayException {
+        return Optional.ofNullable(requestInfo.getPathParameters())
+                   .map(params -> params.get(FILE_IDENTIFIER))
+                   .map(SortableIdentifier::new)
+                   .orElseThrow(() -> new BadRequestException(COULD_NOT_GET_FILE_IDENTIFIER));
```

Comment:
Method is duplicated, what about reusing existing method?
```
    public static SortableIdentifier getFileEntryIdentifier(RequestInfo requestInfo) throws ApiGatewayException {
        var fileIdentifier = getFileIdentifier(requestInfo).toString();
        return new SortableIdentifier(fileIdentifier);
    }
```

Reply:
not possible because UUID will loose the time part of sortableidentifier